src/lib/viewers/doc/_docBase.scss

```diff
@@ -165,6 +165,19 @@ $thumbnail-sidebar-width: 226px;
             display: block;
         }
     }
+
+    &.bp-is-small {
```

Comment:
Is this meant to override the styles in the breakpoint? If so, two levels of overrides can start to get confusing; if not, can it live up at the top of the `.bp` declaration so it's more clear what we're modifying?

Reply:
I removed the breakpoint CSS rules so hopefully this simplifies thing in terms of only having 1 level of override now

Reply:
It's still three levels of CSS, base and two overrides, since the base state is for the components to be hidden. If we turn the CSS class to a positive condition (maybe `.bp-thumbnails-capable`?), then it'd let you keep it to two levels.